go/hyperkit.go

```diff
@@ -428,7 +430,18 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	}
 
 	for _, p := range h.Disks {
-		a = append(a, "-s", fmt.Sprintf("%d:0,virtio-blk,%s", nextSlot, p.Path))
+		// Default the driver to virtio-blk
+		driver := "virtio-blk"
+		if p.Driver == "" {
```

Comment:
D'oh. That's what I get for manually copying the changes back out of my vendor directory...